src/commands/analytics/display-analytics.ts

```diff
@@ -305,16 +406,17 @@ function formatData(
     top_five_alert_types: sortedTopFiveAlerts
   }
 }
+
 function formatDate(date: string): string {
   return `${Months[new Date(date).getMonth()]} ${new Date(date).getDate()}`
 }
 
 function renderLineCharts(
   grid: contrib.grid,
-  screen: any,
+  screen: Widgets.Screen,
   title: string,
-  coords: number[],
-  data: { [key: string]: number }
+  coords: Array<number>,
```

Comment:
☝️ Added a lint rule to add consistency to types (it's a split the difference approach)